packages/zeebe-element-templates-json-schema/src/defs/icon.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$id": "#/icon",
+  "type": "object",
+  "description": "Custom icon to be shown on the element",
+  "default": {},
+  "properties": {
+    "contents": {
+      "$id": "#/icon/contents",
+      "type": "string",
+      "description": "The svg contents of an icon.",
+      "pattern": "^(data:)(?<mime>[\\w/\\-\\.]+)",
```

Comment:
I'm wondering why we don't settle down on:

* this can be a valid, safe `image#src` reference. This includes data-uri (including mime type, but also encoding) + relative or absolute http / https URIs.).

Reply:
We should account for encoding to be present, i.e.:

```
data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEAAAAALAAAAAABAAEAAAIBAAA=
```

I'd love to see a relevant test.

But generally speaking: I'd be happy to relax the rules for the moment to just verify this is a `data:` or `http(s):` protocol URI.

Reply:
Let's relax the pattern for the moment and just make sure that

> verify this is a data: or http(s): protocol URI.